about-us.php

```diff
@@ -1,98 +1,75 @@
-                        <br>he has successfully lead, and developed dozens of media projects.
-                        <br>Most importantly he specializes in pre and post production.   
-                    </p>
-
-        </div>
-        <div>
-
-                <h2>Waymon Brown - Account Executive</h2>
-                    
-                    <p>
-                        <br>Waymon entered the media field several years ago as a freelance web developer,
-                        <br>while emerged into business management for actors. Some of whom have appeared 
-                        <br>on nationally syndicated television shows. 
-                        <br>He has also worked with local organizations to help them use technology 
-                        <br>in their practices. To be more productive and cost effective in their projects.
-                    </p>
-        </div>
-
-
-
-</body>
+            <p>Add more to this blurb...</p>
+        </div>   
+   
+        <div name="cardGallery">
+            <div class="row">
+                <div class="col-sm-6">
+                    <div class="card">
+                        <img src="..." class="card-img-top" alt="...">
+                        <div class="card-body">
+                            <h5 class="card-title">Special title treatment</h5>
+                            <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+                            <a href="#" class="btn btn-primary">Go somewhere</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6">
+                    <div class="card">
+                        <img src="..." class="card-img-top" alt="...">
+                        <div class="card-body">
+                            <h5 class="card-title">Special title treatment</h5>
+                            <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+                            <a href="#" class="btn btn-primary">Go somewhere</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+         </div>
+   
+   
+   </div>
+
+
+<footer class="footer"></footer>        
+
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```

Comment:
whats with these scripts? is it the individual links or two of the bundle and whats with the version? Me thinks this is wrong...

Reply:
Resolved with commit #c69386c.